cmake-edit-core/src/main/java/ru/urururu/cmakeedit/core/checker/SimulationState.java

```diff
@@ -51,82 +140,16 @@ public void accept(CommandInvocationNode node) {
             }
 
             private void processUsages(CommandInvocationNode node) {
-                if (node.getCommandName().equals("set")) {
-                    List<Node> arguments = node.getArguments();
-
-                    if (arguments.isEmpty()) {
-                        return;
-                    }
-
-                    arguments.get(0).visitAll(new NodeVisitorAdapter() {
-                        @Override
-                        public void accept(ExpressionNode node) {
-                            processExpression(node, suspiciousPoints);
-                        }
-                    });
-
-                    for (int i = 1; i < arguments.size(); i++) {
-                        arguments.get(i).visitAll(new NodeVisitorAdapter() {
-                            @Override
-                            public void accept(ExpressionNode node) {
-                                processExpression(node, suspiciousPoints);
-                            }
-
-                            @Override
-                            public void accept(ArgumentNode node) {
-                                // not to hardcore?
-                                String argument = node.getArgument();
-                                processUsage(argument, suspiciousPoints);
-                            }
-                        });
-                    }
+                List<Node> arguments = node.getArguments();
 
-                    return;
+                for (Node argument : arguments) {
+                    String arg = getValue(argument);
+                    processUsage(arg); // todo shouldnt do for all!
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1135)